### PR TITLE
chore(database): add aiven_extrs extension to postgres docker

### PR DESCRIPTION
**Pull request by alemagio**
## Description & motivation

- Created a build step for postgres to compile aiven_extras
- Created a migration to enable the extension  the database

## Checklist:

- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

### CODE REVIEW

File: docker/postgres/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    'llvm13=~13.0'
+
+RUN git clone --branch 1.1.9 https://github.com/aiven/aiven-extras.git aiven-extras \
+  && cd $WORKDIR/aiven-extras \
+  && make \
+  && make install
+
+FROM postgres:14.5-alpine
+
+COPY --from=builder /aiven-extras/aiven_extras.control /usr/local/share/postgresql/extension/aiven_extras.control
```
[the remark this answers is not in the document]
Comment (alemagio) [reply docker/postgres/Dockerfile:18]: The problem is that this extension is not even in the Postgres registry and must be compiled manually.
As far as I understood we can create a ticket request for DO to support this extension in managed databases but we would still need these changes (or something similar) for our local dev env.

**Comment by gjedlicska** [comment]
@alemagio the migration might be a bit premature. Most of the team will not have the need to rebuild their DB-s right away. We'll also need to double check with the production DB-s.

Without that, rolling out the migration to DB-s without the aiven_extras plugin installed, will fail.

I'm creating a ticket, so that we do not forget this

**Review by gjedlicska — CHANGES_REQUESTED** [review]
im currently running into this issue, when trying to create a logical replication:

`logical decoding requires wal_level >= logical`

This wasn't part of the og task spec, but without that setting, the subscriptions do not work

**Comment by alemagio** [comment]
> im currently running into this issue, when trying to create a logical replication:
> 
> `logical decoding requires wal_level >= logical`
> 
> This wasn't part of the og task spec, but without that setting, the subscriptions do not work

I'll try it and see, if I can find a quick fix I'll add to this ticket, otherwise I'll create a dedicated one.

**Comment by alemagio** [comment]
> im currently running into this issue, when trying to create a logical replication:
> 
> `logical decoding requires wal_level >= logical`
> 
> This wasn't part of the og task spec, but without that setting, the subscriptions do not work

@gjedlicska I tried and to create a subscription we need to set the system level to `logical` 
```sql
ALTER SYSTEM SET WAL_LEVEL = LOGICAL;
```
after running this we restart the postgres container postgres is now in subscription mode

**Pull request merged**